charts/plex-media-server/templates/statefulset.yaml

```diff
@@ -119,6 +119,10 @@ spec:
           value: compute,video,utility
         {{- end }}
         {{- with .Values.pms.resources }}
+        livenessProbe:
```

Comment:
It appears to me that this is incorrectly nested within resources due to the above line.